config/base/manager/manager.yaml

```diff
@@ -22,6 +22,18 @@ spec:
         - --enable-leader-election
         image: ghcr.io/banzaicloud/kafka-operator:latest
         name: manager
+        readinessProbe:
+          initialDelaySeconds: 20
+          periodSeconds: 15
+          httpGet:
+            port: 8081
+            path: /readyz
+        livenessProbe:
+          initialDelaySeconds: 15
+          periodSeconds: 10
+          httpGet:
+            port: 8081
+            path: /healthz
```

Comment:
**Optional**: In charts/kafka-operator/templates/operator-deployment-with-webhook.yaml the order was reversed, we **MAY** want to be consistent in both places, logically readiness, liveness is favored, alphabetically liveness readiness is favored. I can live with both, has my preference but leave it to you.

Reply:
I agree.